source/common/fileName.js

```diff
@@ -0,0 +1,7 @@
+import moment from 'moment';
```

Comment:
@daniloprates please add `// @flow` to the top of this file.

Reply:
✔︎